src/styles/globals.css

```diff
@@ -393,24 +393,8 @@ th {
     color: rgba(108, 117, 125, 0.4);
 }
 
-.form-control {
-    border-radius: calc(0.25rem - 0px) calc(0.25rem - 0px) calc(0.25rem - 0px) calc(0.25rem - 0px);
-    border-color: #e7e7ed;
-    box-shadow: none;
-    border-width: 1px;
-    border-style: solid;
-    font-size: 0.875rem;
-    display: block;
-    transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
-    font-weight: 400;
-    height: 1.5rem;
-    padding-bottom: 0.25rem;
-    padding-left: 0.75rem;
-    padding-right: 0.75rem;
-    padding-top: 0.25rem;
-}
```

Comment:
Why remove entire form.control ? This is to be exactly like liferay styling, incluDing the orange fading in.

Reply:
@heliocastro This snippet made the input look smaller. I am not sure why we need this. I think bootstrap's default  is enough

With form-control
![image](https://github.com/eclipse-sw360/sw360-frontend/assets/94659621/f9ce7545-68f6-492b-b279-d44872a92d43)

Without form-control
![image](https://github.com/eclipse-sw360/sw360-frontend/assets/94659621/55b070a1-0eae-4653-919d-fc84b1f295dd)

Reply:
Ok, got the point